woocommerce-gateway-ebanx/gateways/class-wc-ebanx-debit-card-gateway.php

```diff
@@ -87,7 +93,7 @@ public function payment_fields()
 	protected function request_data($order)
 	{
 		if (empty($_POST['ebanx_debit_token']) || empty($_POST['ebanx_billing_cvv'])) {
```

Comment:
We need to take careful with this function yet... Because many method are breaking when we change to the `WC_EBANX_Request`. Please, test in everywhere before change it..